lib/taurus/cli/common.py

```diff
@@ -0,0 +1,97 @@
+#############################################################################
+##
+# This file is part of Taurus
+##
+# http://taurus-scada.org
+##
+# Copyright 2011 CELLS / ALBA Synchrotron, Bellaterra, Spain
+##
+# Taurus is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+##
+# Taurus is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+##
+# You should have received a copy of the GNU Lesser General Public License
+# along with Taurus.  If not, see <http://www.gnu.org/licenses/>.
+##
+#############################################################################
+
+import click
+
+
+log_port = click.option(
```

Comment:
I am not sure if this option is common enough to be included here.

IMHO, only options that are likely to be ever imported from different packages should be included here because, by including them, we give them the status of "official API" (and therefore we assume the responsibility of maintaining them).

Note: if an option is likely to be used by several commands but only from a given specific submodule, maybe it is more reasonable to implement them in that specific submodule.

Reply:
Key for using common options was for me only that option is used in more than one place. However, I see your point and I agree that this can be exposed as external interface. Shall I create `cli` module inside module which uses those common options and put it there? This would make codebase DRY.

Reply:
> Shall I create cli module inside module 

if the module using it deserves a separate file, then yes. If tha module that uses it is simple, maybe creating a separate file is not worth it, and the cli options can be implemented in the module itself